com.unity.render-pipelines.core/Editor/Lighting/ProbeVolume/ProbeGIBaking.cs

```diff
@@ -454,7 +452,7 @@ static void OnAdditionalProbesBakeCompleted()
             m_BakedCells.Clear();
 
             // Clear baked data
-            Clear();
+            ProbeReferenceVolume.instance.Clear();
```

Comment:
Why changed to this clear? We nmeed to invalidate data on probe volumes and per scene data too.

Reply:
Indeed there was an error that should be fixed by commit 668245b
My issue is that InvalidateAllAssets was clearing the assets map, so we would loose all references, now i only do AddPendingAssetRemoval.
In Clear though, i still clear the map, and additionally i also deleted the asset file on disk, i find that more logical